internal/command/test.go

```diff
@@ -275,98 +336,187 @@ func (c *TestCommand) ExecuteTestRun(ctx *terraform.Context, run *moduletest.Run
 	return state
 }
 
-func (c *TestCommand) GetInputValues(locals map[string]hcl.Expression, globals map[string]hcl.Expression, config *configs.Config) (terraform.InputValues, tfdiags.Diagnostics) {
-	variables := make(map[string]hcl.Expression)
-	for name := range config.Module.Variables {
-		if expr, exists := locals[name]; exists {
-			// Local variables take precedence.
-			variables[name] = expr
-			continue
-		}
+// execute executes Terraform plan and apply operations for the given arguments.
+//
+// The command argument decides whether it executes only a plan or also applies
+// the plan it creates during the planning.
+func (runner *TestRunner) execute(run *moduletest.Run, file *moduletest.File, config *configs.Config, state *states.State, opts *terraform.PlanOpts, command configs.TestCommand) (*terraform.Context, *plans.Plan, *states.State, tfdiags.Diagnostics) {
+	if opts.Mode == plans.DestroyMode && state.Empty() {
```

Comment:
Not sure if it matters here, but `Empty` also looks for rot module outputs, because in a normal context those need to be removed explicitly by a plan, but for tests maybe it's useful to skip those?

Reply:
Thanks! That's good context for me in the future. I think in this case it's okay, as it'll just go and do a very quick destroy operation and seeing that fail/succeed might be something the user wants to include in the tests.